src/Drawer/Drawer.js

```diff
@@ -71,7 +71,7 @@ export default class Drawer extends Component {
     styleManager: PropTypes.object.isRequired,
   };
 
-  getSlideDirection(anchor) {
+  static getSlideDirection(anchor) {
```

Comment:
What's better in your opinion? A `static` method or just a function in the module? It doesn't need to be part of any public API